dashboards/ansible-tower/ansible-tower.json

```diff
@@ -0,0 +1,801 @@
+                "ignoreTimeRange": false
+              }
+            }
+          },
+          {
+            "title": "Ansible Tower Log",
+            "layout": {
+              "column": 1,
+              "row": 16,
+              "width": 12,
+              "height": 3
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "logger.log-table-widget"
+            },
+            "rawConfiguration": {
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "FROM Log SELECT *"
+                }
+              ]
+            }
+          }
+        ]
+      },
+      {
+        "name": "Infrastructure",
+        "description": null,
+        "widgets": [
+          {
+            "title": "CPU usage (%)",
+            "layout": {
+              "column": 1,
+              "row": 1,
+              "width": 4,
+              "height": 3
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.line"
+            },
+            "rawConfiguration": {
+              "config": {
+                "bisect": {
+                  "position": null
+                }
+              },
+              "legend": {
+                "enabled": false
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": "3510613",
+                  "events": {
+                    "agent_ids": [
+                      "8263907339200612017"
+                    ]
+                  },
+                  "query": "SELECT average(cpuPercent) AS 'CPU used %' FROM SystemSample WHERE `entityGuid` = 'MzUxMDYxM3xJTkZSQXxOQXw4MjYzOTA3MzM5MjAwNjEyMDE3' TIMESERIES auto"
+                }
+              ],
+              "yAxisLeft": {
+                "max": 100,
+                "min": 0,
+                "zero": null
+              }
+            }
+          },
+          {
+            "title": "Memory usage (%)",
+            "layout": {
+              "column": 5,
+              "row": 1,
+              "width": 4,
+              "height": 3
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.line"
+            },
+            "rawConfiguration": {
+              "config": {
+                "bisect": {
+                  "position": null
+                }
+              },
+              "legend": {
+                "enabled": false
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": "3510613",
+                  "events": {
+                    "agent_ids": [
+                      "8263907339200612017"
+                    ]
+                  },
+                  "query": "SELECT average(memoryUsedPercent) AS 'Memory used %' FROM SystemSample WHERE `entityGuid` = 'MzUxMDYxM3xJTkZSQXxOQXw4MjYzOTA3MzM5MjAwNjEyMDE3' TIMESERIES auto"
```

Comment:
We should remove `entityGuid` here.

Reply:
done.. removed it and updated the dashboard json